client-react/src/pages/app/functions/function/function-log/FunctionLogAppInsightsDataLoader.tsx

```diff
@@ -256,12 +258,26 @@ const FunctionLogAppInsightsDataLoader: React.FC<FunctionLogAppInsightsDataLoade
     setAllLogEntries([]);
   };
 
+  const getShowFilteredLogsMessage = () => {
+    return (
+      !!Url.isFeatureFlagEnabled(CommonConstants.FeatureFlags.useNewFunctionLogsApi) &&
```

Comment:
`isFeatureFlagEnabled` already returns a Boolean.

```suggestion
      Url.isFeatureFlagEnabled(CommonConstants.FeatureFlags.useNewFunctionLogsApi) &&
```